CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.1.0
```

Comment:
I think that we should omit the `CHANGELOG` from this PR and update it during the Git Flow release creation process.